pkg/api/defaults-pod-security-policy.go

```diff
@@ -0,0 +1,21 @@
+package api
```

Comment:
I don't think we need this. We already have pep YAML files with placeholders. They should suffice as defaults